src/Aspire.Hosting/Utils/VolumeNameGenerator.cs

```diff
@@ -5,16 +5,26 @@
 
 namespace Aspire.Hosting.Utils;
 
-internal static class VolumeNameGenerator
+/// <summary>
+/// Utility class for generating volume names
+/// </summary>
+public static class VolumeNameGenerator
 {
+    /// <summary>
+    /// Creates a volume name with the form <c>{applicationName}-{sha256 of apphost path}-{resourceName}-{suffix}</c>, e.g. <c>myapplication-a345f2451-postgres-data</c>.
+    /// </summary>
+    /// <typeparam name="T">The resource type.</typeparam>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="suffix">The suffix to append to the volume name.</param>
+    /// <returns>The volume name.</returns>
+    /// <exception cref="ArgumentException"></exception>
     public static string CreateVolumeName<T>(IResourceBuilder<T> builder, string suffix) where T : IResource
```

Comment:
Probably want to think about naming here.. duplicative ? `VolumeNameGenerator.Generate` ?

Reply:
I have no preference on this, simply changed the visibility.  I agree though, Generate seems more consistent and it’s an opportunity to polish it a bit.  I’ll update it

Reply:
I have updated to change the name from CreateVolumeName to Generate as suggested.

Reply:
@danmoseley Are you good with the changes made?